docs/1.0. Certificate Provisioning.md

```diff
@@ -67,47 +67,48 @@ The entity that is using the NMOS API, for example:
 
 This document is not concerned with the security of the connection used to carry out provisioning of the TLS Certificate, but for the mechanism described in this document to be effective the connection must be secured, ideally using the recommendations covered in [BCP-003-01][].
 
 ## Automated Certificate Provisioning Flow (informative)
 
 To enable zero-configuration TLS Certificate provisioning, NMOS implementations must include a unique TLS Client Certificate, signed by the manufacturer's Certificate Authority. Manufacturers must provide the corresponding Root Certificate Authority public key to customers. If the TLS Client Certificate or the chain of trust is compromised, the manufacturer must revoke the comprised certificates.
 
-1. Before the NMOS Node(EST Client) is shipped from the factory it must be provisioned with a unique TLS Client Certificate, signed by the manufacturer's Certificate Authority
-2. When the EST Client is connected to the target environment's network, it will first discover the location of the EST Server using Unicast DNS-SD.
-3. The EST Client should then request the Root CA for the target network, from the EST Server
-    - The EST Client should assume the EST server found using DNS-SD is trusted and need not perform TLS authentication of the EST Server for this initial transaction.
-    - Using the Root CA returned to secure further communications with the EST Server and NMOS Servers
-4. The EST Client must generate a Certificate Signing Request(CSR) for all supported digital signature algorithms with appropriate key lengths
-    - Included in the CSR must be the DNS resolvable name for the current domain of the EST Client
-    - Appropriate values for the remaining CSR fields should be used
-5. The EST Client should send each CSR to the EST Server
-    - The EST Client must provide the manufacturer issued certificate during the TLS handshake, if valid
-    - The EST Server will use the TLS Client Certificate to authenticate and then authorise whether the EST Client. If the EST Client meets the local CA's security policy, it will be issued with a TLS Server Certificate.
-6. The EST Client should provide the returned TLS Certificate and the corresponding chain of trust in all future responses returned by its NMOS APIs
+1. Before the NMOS Server (EST Client) is shipped from the factory it must be provisioned with a unique TLS Client Certificate, signed by the manufacturer's Certificate Authority.
+2. When the EST Client is connected to the target environment's network, it will first discover the location of the EST Server using unicast DNS-SD.
+3. The EST Client will then request the Root CA for the target network, from the EST Server.
+   - The EST Client can assume the EST server found using DNS-SD is trusted and need not perform TLS authentication of the EST Server for this initial transaction.
```

Comment:
```suggestion
2. When the NMOS Server is connected to the target environment's network, it will first discover the location of the EST Server using unicast DNS-SD.
3. The NMOS Server will then request the Root CA for the target network, from the EST Server.
   - The NMOS Server can assume the EST server found using DNS-SD is trusted and need not perform TLS authentication of the EST Server for this initial transaction.
```